### PR TITLE
Mohbeen week 5

### PR DESCRIPTION
**Pull request by mohbeen29**

### CODE REVIEW

File: week-5-practise/js/main.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
+/*### part 1
+
+whhen clicking **blue** it should change:
+        - **Jumbotron** background color to `#588fbd`
+        - **Donate a bike** button background color to `#ffa500`
+        - **Volunteer** button background color to `black` and text color to `white`*/
+
+var blueTheme = document.querySelector("#blueBtn");
+var myContainer = document.querySelector(".jumbotron");
+var myButton = document.querySelectorAll(".btn-lrg");
```
Comment (foaly-nr1) [inline week-5-practise/js/main.js]: These are all right! Just a minor detail: variables assigned an array should have a plural name, e.g. `myButtons`.

File: week-5-practise/js/main.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+}
+
+/*When clicking **orange** it should change:
+        - **Jumbotron** background color to `#f0ad4e`
+        - **Donate a bike** button background color to `#5751fd`
+        - **Volunteer** button background color to `#31b0d5` and text color to `white`*/
+
+var orangeTheme = document.querySelector("#orangeBtn");
+var myContainer = document.querySelector(".jumbotron");
+var myButton = document.querySelectorAll(".btn-lrg");
```
Comment (foaly-nr1) [inline week-5-practise/js/main.js]: You could have saved yourself some of these queries and assignments here as you've already got them above.

File: week-5-practise/js/main.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+- Display an alert to thank you for filling out the form
+    - Blank out(make empty) all the text fields
+
+        ** Important hint:** In your function that handles clicks on the`Submit` button you will need to call`event.preventDefault()` to stop the browser from refreshing the page.To read more on how to do this: https://developer.mozilla.org/en/docs/Web/API/Event/preventDefault 
+*/
+var submitBtn = document.querySelector("#submitBtn");
+var controlForm = document.querySelectorAll(".form-control");
+var email = document.querySelector("#exampleInputEmail1");
+var name2 = document.querySelector("#name-input");
+var describe = document.querySelector("#exampleTextarea");
```
Comment (foaly-nr1) [inline week-5-practise/js/main.js]: I find the variables `submitBtn` and `controlForm` self-explanatory, but struggle with `email` and `name2`. I think the variable names suggest that you probably have string values in them. Maybe change them to `emailInput` and `nameInput`?

File: week-5-practise/js/main.js [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+    name2.style.backgroundColor = "red";
+  }
+  if (describe.value == "") {
+    describe.style.backgroundColor = "red";
+  }
+  if (
+    email.value.length > 0 &&
+    name2.value.length > 0 &&
+    describe.value.length > 0 &&
+    email.value.includes("@")
```
Comment (foaly-nr1) [inline week-5-practise/js/main.js]: Well done 👏🏻The only thing missing is changing the red background colour back to normal once the form has been submitted successfully.

File: week-5-practise/js/main.js (hunk 2 above)
Comment (mohbeen29) [reply week-5-practise/js/main.js]: how? I wanted to but dint know how. and without it, it didnt work

File: week-5-practise/js/main.js (hunk 4 above)
Comment (mohbeen29) [inline week-5-practise/js/main.js]: im still unsure how this last bit just
converts to blank? it assigns to value "" (0) but where the actual function to convert it?

im talking about after the alert bit btw below.

Comment (mohbeen29) [reply week-5-practise/js/main.js]: i will try add that thank you

File: week-5-practise/js/main.js (hunk 2 above)
Comment (foaly-nr1) [reply week-5-practise/js/main.js]: You could for example delete lines 28-30 and nothing should change as those 3 variables were declared in lines 10-12.

**Pull request closed without merging**